### PR TITLE
Solved #47 issue.

### PR DESCRIPTION
**Pull request by munnokd**
## Resolves:

#47

### CODE REVIEW

**Comment by AmanNegi** [comment]
I guess, It would be better if we can link to the latest release as well.

**Comment by munnokd** [comment]
> I guess, It would be better if we can link to the latest release as well.

I don't understand what are you trying to say.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,12 @@
 + The platform provides **secure payment options, shipping services, and customer service** to ensure *safe and efficient transactions. *
 + With **AgroMillets**, farmers can expand their customer base and customers can support sustainable farming practices while enjoying a wider variety of products.
 
+
+## 🚀 Live Preview 
++ **Website**: [`AgroMillets.com`](https://agro-millets.netlify.app/)
++ **App**: Under Releases
```
[the remark this answers is not in the document]
Comment (munnokd) [reply README.md:18]: Okay bro I will add it.

**Comment by munnokd** [comment]
Is this okay @AmanNegi bro?

**Comment by AmanNegi** [comment]
Great! Looks good. Thanks a lot for contributing ☺

**Pull request merged**